apps/antalmanac/src/components/Calendar/Toolbar/EditSchedule/ScheduleNameDialog.tsx

```diff
@@ -49,7 +49,7 @@ const ScheduleNameDialog = forwardRef((props: ScheduleNameDialogProps, ref) => {
 
     // We need to stop propagation so that the select menu won't close
     const handleOpen = useCallback(
-        (event: React.MouseEvent) => {
+        (event: MouseEvent<HTMLLIElement>) => {
```

Comment:
Strongly typing the generic when you don't need it is just superfluous.

```suggestion
        (event: MouseEvent) => {
```